examples/stable_diffusion_xl/modelarts/fmk.py

```diff
@@ -0,0 +1,94 @@
+import os
+import pathlib
+import subprocess
+from contextlib import contextmanager
+
+from common import ModelArts, RankTableEnv, RunAscendLog
+
+log = RunAscendLog.get_run_ascend_logger()
+
+
+class FMK:
```

Comment:
Use the full name instead of the abbreviation?

Reply:
This is the official information downloaded by modelarts.